src/main/java/com/stripe/model/PaymentIntent.java

```diff
@@ -1141,6 +1141,20 @@ public PaymentIntent capture(PaymentIntentCaptureParams params, RequestOptions o
       /** The base64 image data for a pre-generated QR code. */
       @SerializedName("image_data_url")
       String imageDataUrl;
+
+      /**
+       * The image_url_png string used to render QR code, can be used as &lt;img src=&quot;…&quot;
```

Comment:
Is the `<`/`>`/`"` replacement easy to fix? Or is this expected?

Reply:
Similar to .NET, we need to encode these since otherwise the Javadoc interpreter will try to interpret this comment and we're giving this as a code snippet (ie. we want users to explicitly see `<img src="...">` and not a rendered image).